src/test/resources/producer/chips-rest-interfaces-send.avsc

```diff
@@ -0,0 +1,31 @@
+{
+  "type": "record",
+  "namespace": "chips",
```

Comment:
The namespace and name has been updated in latest version of kafka-schemas

Reply:
Updated